commerce-configuration-1/.generated/actions/app-management/get-configuration.js

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+// This file is auto-generated by the pre-app-build hook
+// Do not modify this file directly
+
+import { init } from '@adobe/aio-commerce-lib-config';
+import { 
+  badRequest, 
+  internalServerError, 
+  ok 
+} from '@adobe/aio-commerce-sdk/core/responses';
+
+/**
+ * Get the configuration.
+ * @param params - The input parameters.
+ * @returns The response object containing the configuration data.
+ */
+export async function main(params) {
+  try {
+    const config = init();
+    const id = params.id
+    const code = params.code
+    const level = params.level;
+
+    if (!id && !code) {
+      return badRequest({ 
+        body: { 
+          code: 'INVALID_PARAMS', 
+          message: 'Either id or code query param is required'
+        }
+      });
+    }
+
+    let appConfiguration;
+    if (id) {
+      appConfiguration = await config.getConfiguration(id);
+    } else if (level) {
+      appConfiguration = await config.getConfiguration(code, level);
+    } else {
+      appConfiguration = await config.getConfiguration(code);
```

Comment:
I haven't seen the implementation of `config.getConfiguration`, but it's strange to me that the same method can behave differently based on the type of arguments provided. It looks like it can be 2 or even three different methods.
```
config.getById(id);
config.getByCode(code);
config.getByCodeAndLevel(code, level);
```
Methods should clearly show what kind of input they expect.